test/TestProjects/AmbiguousMethods-LowLevel/readme.md

```diff
@@ -0,0 +1,26 @@
+# Resolving Ambiguous Methods In LowLevel
+
+This test project focuses on testing the generator's capability to resolve ambiguous methods in low level.
+
+The scenario is that when growing a DPG client into an exsiting HLC client, there could be ambiguous methods between existing client and the DPG client to generate. The generator should have the capability to detect and resolve some ambiguous caes automatically.
+
+See MethodAmbiguityResolver.cs for details.
+
+
+## Structure
+
+- The two clients under the root folder are simulation of HLC clients.
```

Comment:
Do you mean these are the partial classes that add the convenience APIs to each of the clients?  (I was at first confused thinking these were the legacy-generated RestClients used by HLC libraries)

Reply:
@annelo-msft No. Those are the simulation of existing codes.